src/utilities/include/par.h

```diff
@@ -146,29 +139,54 @@ template <typename DerivedPolicy, typename InputIterator1,
 OutputIterator copy_if(ExecutionPolicy policy, InputIterator1 first,
                        InputIterator1 last, OutputIterator result,
                        Predicate pred) {
-#if MANIFOLD_PAR == 'T'
-  if (policy == ExecutionPolicy::Seq)
-    return std::copy_if(first, last, result, pred);
-  else
-    return std::copy_if(std::execution::par_unseq, first, last, result, pred);
-#else
+  // #if MANIFOLD_PAR == 'T'
+  //   if (policy == ExecutionPolicy::Seq)
+  //     return std::copy_if(first, last, result, pred);
+  //   else
+  //     return std::copy_if(std::execution::par_unseq, first, last, result,
```

Comment:
Also, it appears LLVM is missing copy_if for PSTL in general, not just the stencil version.